dist/images/ovnkube.sh

```diff
@@ -398,6 +402,24 @@ check_health() {
     ;;
   esac
 
+  if $DB_CLUSTERED; then
+    if [[ ${1} == "ovnnb_db" ]]; then
```

Comment:
so as long as ovsdb-server for NB/DB exits (E.g. coredump), we wipe out the DB file? This is quite intrusive.

Reply:
no this was only intended to wipe the db if we are no longer part of the cluster, to force rejoin with a clean slate, then the raft cluster woudl resync our db. However, I agree that this is probably too aggressive. Instead of just checking if the command doesn't return 0, we can check if the command returns 2 and "Unknown" which will signal to us that the node is no longer in a cluster.